pkg/controller/installation/typha_autoscaler.go

```diff
@@ -39,16 +39,14 @@ const (
 
 // typhaAutoscaler periodically lists the nodes and, if needed, scales the Typha deployment up/down.
 // The number of Typha replicas depends on the number of nodes:
-// Nodes       Replicas
-//     1              1
-//     2              2
-//     3              3
-//   250              4
-//   500              5
-//  1000              6
-//  1500              7
-//  2000              8
-//  2000+            10
+//   Nodes          Replicas
+// <=    2                 1
+// <=    5                 2
+// <=  100                 3
```

Comment:
I chatted with @caseydavenport a bit about this and we need to stick to 3 when 3 are possible. It is what is documented as suggested. Without pulling in more people and discussing with a wider audience we should have 3 when 3 are available.